NSS-Day-Cohort-53/tabloidcli-refreshing-rhinoceroses
# refactored for sql cascade delete

#### Changes Made
1. Added file `TabloidCLI02.sql` to `root` directory.
2. Modified file `AuthorRepository.cs, BlogRepository.cs, PostRepository.cs` to remove `unnecessary deletes from query`.
3. Modified file `AuthorManager.cs and BlogManager.cs` to remove `unnecessary delete calls`

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout src-cascade
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
4. Test app functionality.
    > Make a blog with tags that has a post with tags and notes and delete it
    > Make a author with tags that has a post with tags and notes and delete it
5. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
